### PR TITLE
Adderess management

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
-added payment gateway
-added address management
-added order summary

### CODE REVIEW

File: src/components/address/Address.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { ACTION_TYPE } from "../../store/Actions";
+import { useOrder } from "../../store/data";
```
Comment (ScoobyDubeyDo) [inline src/components/address/Address.jsx:3]: **suggestion**: use barrel export

File: src/components/cart/Cart.jsx [hunk 2]
```diff
@@ -38,13 +38,19 @@ function Cart() {
...
   useEffect(() => {
     const data = calculateTotalSummary(cartState.cart);
     setTotalSummary(data);
   }, [cartState.cart]);
-
+  
+  const cartCheckout=()=>{
+    navigate("/checkout")
+    orderDispatch({type:ACTION_TYPE.PRICE_DETAILS,payload:totalSummary})
+    // orderDispatch({type:ACTION_TYPE.ADDRESS_DETAILS,payload:addre})
```
Comment (ScoobyDubeyDo) [inline src/components/cart/Cart.jsx:52]: **must**: delete the unused code

File: src/components/ordersummary/OrderSummary.jsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { useOrder } from "../../store/data/OrderContext";
+
+function OrderSummary() {
+  const { orderState } = useOrder();
+  const navigate = useNavigate();
+  const { orderedProducts, amount, paymentId, delivery } =
+    orderState.orderDetails;
+  //   console.log(orderState.orderDetails);
```
Comment (ScoobyDubeyDo) [inline src/components/ordersummary/OrderSummary.jsx:10]: **must** delete the comments

**Review by ScoobyDubeyDo — APPROVED** [review]

**Review by muskaanshah — APPROVED** [review]
Looks good to merge, clean understandable code

**Pull request merged**